ldk/javascript/examples/self-test-loop/src/tests/whisper/utils.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  JustifyContent,
+  Direction,
+  WhisperComponentType,
+  Whisper,
+  Component,
+} from '@oliveai/ldk/dist/whisper/types';
+
+export const resolveOnClick = (
+  error: Error,
+  whisperToClose: Whisper,
+  resolve: (value: boolean) => void,
+  reject: (reason?: Error) => void,
+): void => {
+  if (error) {
+    console.error(error);
+    reject(error);
+  }
+  whisperToClose.close(() => {
+    // do nothing.
+  });
+  resolve(true);
+};
+
+export const rejectOnClick = (
+  error: Error,
+  whisperToClose: Whisper,
+  reject: (reason?: Error) => void,
+): void => {
+  if (error) {
+    console.error(error);
+    reject(error);
+  }
+  whisperToClose.close(() => {
+    // do nothing.
+  });
+  reject(new Error('Not rendered correctly.'));
+};
+
+export const resolveRejectButtons = (
```

Comment:
This is very similar to `confirmOrDeny` inside of `tests/whisper/whisper-update.ts`. Not necessarily your issue to combine these but perhaps we can at some point (use the one you created in `/utils`)

Reply:
Yes the idea was to extract this buttons into common place. I just didnt refactored  the whisper update yet.